core/local/index.js

```diff
@@ -13,7 +13,7 @@ const logger = require('../logger')
 const { isUpToDate } = require('../metadata')
 const { hideOnWindows } = require('../utils/fs')
 const sentry = require('../sentry')
-const Watcher = require('./watcher')
+const watcher = require('./watcher')
```

Comment:
Why the lower case here while you haven't changed the case in other files?

Reply:
With an upper-case, flow complains. I needs to manually import the `Watcher` type: L31 `import type { Watcher } from './watcher'`, and it works only if I don't use `Watcher` here too.

So, it's probably just my flow-fu that is not good enough :p